taskt/UI/Forms/Supplement Forms/frmImageCapture.cs

```diff
@@ -1,36 +1,32 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
 using System.Drawing;
 using System.Drawing.Drawing2D;
 using System.Drawing.Imaging;
 using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows.Forms;
 
 namespace taskt.UI.Forms.Supplement_Forms
 {
     public partial class frmImageCapture : Form
     {
-        public frmImageCapture()
-        {
-            InitializeComponent();
-        }
         //These variables control the mouse position
-        int selectX;
-        int selectY;
-        int selectWidth;
-        int selectHeight;
+        private int _selectX;
+        private int _selectY;
+        private int _selectWidth;
+        private int _selectHeight;
         public Pen selectPen;
         public Bitmap userSelectedBitmap;
```

Comment:
Please use PascalCase for the public properties `selectPen` and `userSelectedBitmap`

Reply:
Done